### PR TITLE
Component for adding/selecting organization for contact

### PR DESCRIPTION
**Pull request by harish86**
Screenshot:

Auto suggest:

Adding new organization:

### CODE REVIEW

File: web/controllers/api/v2/organization_controller.ex [hunk 1]
```diff
@@ -28,6 +28,23 @@ defmodule CercleApi.APIV2.OrganizationController do
...
+        where: ilike(o.name, ^"#{q}%"),
+        order_by: o.name
+      organizations = Repo.all(query)
+    else
+      organizations = []
+    end
+
+    render(conn, "index.json", organizations: organizations)
+  end
+
```
Comment (parallel588) [inline web/controllers/api/v2/organization_controller.ex]: Hi @harish86 ,may be better use two method instead use if - else.
like this:
```
def search(conn, %{"q" => q}) do
.. you code .. 
end
def search(conn, _) do
render(conn, "index.json", organizations: [])
end
```

**Comment by harish86** [comment]
@parallel588 I am working on adding tests for this feature. When I run "mix test" it just remains silent, and doesn't output anything. I have installed "chromedriver" and there are no dependency errors. Am I missing something?

File: web/controllers/api/v2/organization_controller.ex (hunk 1 above)
Comment (harish86) [reply web/controllers/api/v2/organization_controller.ex]: @parallel588 Ok

**Comment by parallel588** [comment]
@harish86 could you try run ``` mix test --trace``` .

**Comment by harish86** [comment]
@parallel588 I tried with --trace. It still doesn't output anything. I can see it does compiling and internally it laucnhes chromedriver, but doesn't run the tests. I am on ubuntu.

**Comment by parallel588** [comment]
try enable logs for test env. as here https://github.com/cerclecrm/cercle/blob/master/config/dev.exs#L28

File: web/controllers/api/v2/organization_controller.ex (hunk 1 above)
Comment (harish86) [reply web/controllers/api/v2/organization_controller.ex]: @parallel588 I think with this url format "/organizations/search/:q" we can ignore the else part. Make sense?

Comment (antoineherzog) [reply web/controllers/api/v2/organization_controller.ex]: LGTM @parallel588  ?

Comment (parallel588) [reply web/controllers/api/v2/organization_controller.ex]: i think we can  ignore it, but also better if we adds test (controller) for search methods for both case. with q and when qis empty

**Pull request merged**